src/functions/template-tags/general.php

```diff
@@ -844,11 +844,11 @@ function tribe_event_featured_image( $post_id = null, $size = 'full', $link = tr
 			$post_id = get_the_ID();
 		}
 
-		$image_html     = get_the_post_thumbnail( $post_id, $size );
+		$image_html     = get_the_post_thumbnail( $post_id, apply_filters( 'tribe_event_featured_image_size', $size ) );
```

Comment:
Yes, good catch! For the post thumbnail filter it would result in the following, because we need to pass two args to `the get_the_post_thumbnail` function I need to rely on `call_user_func_array`. Is there another (better) way to do it? 

```php
$image_html = call_user_func_array( 'get_the_post_thumbnail', apply_filters( 'tribe_event_featured_image_size', $post_id, $size ) );

function tribe_event_featured_image_size($post_id, $size) {
    return [$post_id, 'full'];
}
```